dashboard/db/migrate/20160614000000_add_processed_location_to_pd_workshops.rb

```diff
@@ -0,0 +1,20 @@
+class AddProcessedLocationToPdWorkshops < ActiveRecord::Migration
+  def up
+    add_column :pd_workshops, :processed_location, :text, after: :location_address
+
+    Pd::Workshop.reset_column_information
+    Pd::Workshop.find_each do |workshop|
+      begin
+        workshop.processed_location = Pd::Workshop.process_location(workshop.location_address)
+        workshop.save!
+      rescue RuntimeError => e
```

Comment:
Note: this should have been `StandardError`, not `RuntimeError`, to catch all the exceptions. See http://ruby-doc.org/core-2.1.1/Exception.html

@breville encountered a local error due to an invalid workshop that blew up the migration. I just made sure data in prod is valid and ready for the migration.